test/inspector/inspector-helper.js

```diff
@@ -215,10 +215,10 @@ TestSession.prototype.sendInspectorCommands = function(commands) {
       timeoutId = setTimeout(() => {
         let s = '';
         for (const id in this.messages_) {
-          s += id + ', ';
+          s += `${id}, `;
```

Comment:
The same: I would abstain from refactoring too deep: this will make reviewing and testing a lot more difficult.